### PR TITLE
LPS-190464 | Import object entries with site scope

### PR DESCRIPTION
**Pull request by Hijky**
https://liferay.atlassian.net/browse/LPS-201998

### CODE REVIEW

**Comment by Hijky** [comment]
ci:test:headless-functional:nocompile

**Comment by Hijky** [comment]
ci:test:sf

**Comment by Hijky** [comment]
ci:test:headless-functional:nocompile

**Comment by Hijky** [comment]
One of the failures will be fixed in this [ticket](https://liferay.atlassian.net/browse/LPS-203586), the others have a [ticket](https://liferay.atlassian.net/browse/LPS-203459)

**Review by JoyceWang08211 — APPROVED** [review]
LGTM

File: modules/apps/batch-planner/batch-planner-test/src/testFunctional/tests/SupportImportObjectEntriesWithSiteScope.testcase [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+				expected = "Jack");
+		}
+	}
+
+	@priority = 4
+	test CannotCreateSiteScopedObjectEntryWithoutRequiredFields {
+		property portal.acceptance = "true";
+
+		task ("When importing .json file as 'C_Student' Entity Type with default setting") {
+			var scope = TestCase.getSiteName();
```
Comment (yanan) [inline modules/apps/batch-planner/batch-planner-test/src/testFunctional/tests/SupportImportObjectEntriesWithSiteScope.testcase:173]: Can we move this to be the default scope in importFile()?

File: modules/apps/batch-planner/batch-planner-test/src/testFunctional/tests/SupportImportObjectEntriesWithSiteScope.testcase [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+				entityType = "C_Student (v1_0 - Liferay Object REST)",
+				fileName = "site_scope_objectEntry_import_without_required_fields.json",
+				noImport = "true",
+				scope = ${scope});
+
+			Button.click(button = "Next");
+		}
+
+		task ("Then importing the item is prevented") {
+			AssertElementNotPresent(
```
Comment (yanan) [inline modules/apps/batch-planner/batch-planner-test/src/testFunctional/tests/SupportImportObjectEntriesWithSiteScope.testcase:185]: Wondering is there any error message/alert?

**Review by yanan — COMMENTED** [review]
Overall are good, see previous two comments

File: modules/apps/batch-planner/batch-planner-test/src/testFunctional/tests/SupportImportObjectEntriesWithSiteScope.testcase (hunk 2 above)
Comment (Hijky) [reply modules/apps/batch-planner/batch-planner-test/src/testFunctional/tests/SupportImportObjectEntriesWithSiteScope.testcase:185]: what your said it's another test 'ErrorMessageIsShownWhenRequiredFieldsMissing' FP=3

Comment (yanan) [reply modules/apps/batch-planner/batch-planner-test/src/testFunctional/tests/SupportImportObjectEntriesWithSiteScope.testcase:185]: got it, thanks

**Comment by Hijky** [comment]
ci:test:sf

**Comment by Hijky** [comment]
ci:test:headless-functional:nocompile

**Comment by Hijky** [comment]
ci:test:headless-functional:nocompile

**Comment by Hijky** [comment]
ci:test:sf

**Comment by Hijky** [comment]
ci:test:sf

**Comment by Hijky** [comment]
ci:test:headless-functional:nocompile

**Comment by Hijky** [comment]
The failures are about [LPS-203459](https://liferay.atlassian.net/browse/LPS-203459)

**Review by yanan — APPROVED** [review]

**Comment by Hijky** [comment]
ci:forward

**Comment by Hijky** [comment]
ci:forward:force

**Pull request closed without merging**